modules/structs/visible_type.go

```diff
@@ -56,3 +60,10 @@ func ExtractKeysFromMapString(in map[string]VisibleType) (keys []string) {
 	}
 	return keys
 }
+
+func ConvertStringToVisibleType(s string) VisibleType {
```

Comment:
Why?

Reply:
22cdf7e191 I forgot to remove it when I changed the implementation. Thank you!